dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1805,7 +1804,15 @@ public Node visitCompoundAssignment(CompoundAssignmentTree tree, Void p) {
                         assert (kind == Tree.Kind.PLUS_ASSIGNMENT);
                         Node targetRHS = stringConversion(targetLHS);
                         value = stringConversion(value);
-                        Node r = new StringConcatenateAssignmentNode(tree, targetRHS, value);
+                        BinaryTree operTree =
+                                treeBuilder.buildBinary(
+                                        leftType,
+                                        withoutAssignment(kind),
+                                        tree.getVariable(),
+                                        tree.getExpression());
+                        Node operNode = new StringConcatenateNode(operTree, targetRHS, value);
+                        extendWithNode(operNode);
+                        AssignmentNode r = new AssignmentNode(tree, targetLHS, operNode);
```

Comment:
Let's rename it to `assignNode` for consistency

Reply:
Thanks, done.